NLog.Windows.Forms/FormHelper.cs

```diff
@@ -1,165 +1,236 @@
+
+            return null;
+        }
+
+        /// <summary>
+        /// Creates a form.
+        /// </summary>
+        /// <param name="name">Name of form.</param>
+        /// <param name="width">Width of form.</param>
+        /// <param name="height">Height of form.</param>
+        /// <param name="show">Auto show form.</param>
+        /// <param name="showMinimized">If set to <c>true</c> the form will be minimized.</param>
+        /// <param name="toolWindow">If set to <c>true</c> the form will be created as tool window.</param>
+        /// <returns>Created form.</returns>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Globalization", "CA1303:Do not pass literals as localized parameters", MessageId = "System.Windows.Forms.Control.set_Text(System.String)", Justification = "Does not need to be localized.")]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2000:Dispose objects before losing scope", Justification = "Objects are disposed elsewhere")]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA2204:Literals should be spelled correctly", Justification = "Using property names in message.")]
+        internal static Form CreateForm(string name, int width, int height, bool show, bool showMinimized, bool toolWindow)
+        {
+            var f = new Form
+            {
+                Name = name,
+                Text = "NLog",
+                Icon = GetNLogIcon()
+            };
+
+#if !Smartphone
+            if (toolWindow)
+            {
+                f.FormBorderStyle = FormBorderStyle.SizableToolWindow;
+            }
+#endif
+            if (width > 0)
+            {
+                f.Width = width;
+            }
+
+            if (height > 0)
+            {
+                f.Height = height;
+            }
+
+            if (show)
+            {
+                if (showMinimized)
+                {
+                    f.WindowState = FormWindowState.Minimized;
+                    f.Show();
+                }
+                else
+                {
+                    f.Show();
+                }
+            }
+
+            return f;
+        }
+
+        private static Icon GetNLogIcon()
+        {
+            using (var stream = typeof(FormHelper).Assembly.GetManifestResourceStream("NLog.Windows.Forms.Resources.NLog.ico"))
+            {
+                return new Icon(stream);
+            }
+        }
+
+#region Link support
+        /// <summary>
+        /// Replaces currently selected text in the RTB control with a link
+        /// </summary>
+        /// <param name="textBox">target control</param>
+        /// <param name="text">visible text of the new link</param>
+        /// <param name="hyperlink">value of the new link</param>
+        internal static void ChangeSelectionToLink(RichTextBox textBox, string text, string hyperlink)
+        {
+            int selectionStart = textBox.SelectionStart;

```

Comment:
please remove old code and document some about this rtf1 (because you understand this :))